Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -487,11 +480,14 @@ public void FixedFrequencyUpdate(float deltaTime)
             EventActions.Trigger("OnUpdate", this, deltaTime);
         }
 
-        if (IsWorkshop && IsBeingDestroyed == false)
+        if (!IsBeingDestroyed)
```

Comment:
I think that we should probably pass ```IsBeingDestroyed``` to ```FixedFrequencyUpdate()```, or check inside that function since it can access the Furniture. In the future we will have components that will still run after the furniture is set to be destroyed, for example, a component that handles doors.